pkg/api/getambassador.io/v2/crd_mapping.go

```diff
@@ -85,6 +85,10 @@ type MappingSpec struct {
 	// +k8s:conversion-gen=false
 	TLS *BoolOrString `json:"tls,omitempty"`
 
+	// +kubebuilder:validation:MinItems=1
+	// +k8s:conversion-gen:rename=HealthChecks
+	V3HealthChecks []HealthCheck `json:"v3health_checks,omitempty"`
```

Comment:
Can we have this just use `v3alpha1.HealthCheck` instead of having to duplicate types in `v2/common.go`?